mod/filter/set/abstract/browse_filter_form.rb

```diff
@@ -45,19 +31,17 @@ def target_type_id
   WikirateCompanyID
 end
 
-def get_query params={}
-  search_args = filter_wql
-  add_sort_wql search_args, sort_param if sort?
-  params[:query] = search_args
-  super(params)
+def wql_hash
+  wql = filter_wql
+  wql[:limit] = 20
+  add_sort_wql wql, sort_param if sort?
+  wql
```

Comment:
I got a lot of "no limit" exceptions im `item_cards` and `item_names` calls. That was the easiest way to fix that